src/core/binding.c

```diff
@@ -1401,13 +1401,8 @@ QuicBindingDeliverDatagrams(
     // be used for partitioning the look up table.
     //
     // For long header packets for server owned bindings, the packet's DestCid
-    // was not necessarily generated locally, so cannot be used for routing.
-    // Instead, a hash of the tuple and source connection ID (SourceCid) is
-    // used.
-    //
-    // The exact type of lookup table associated with the binding varies on the
```

Comment:
Removed this para because it seems to be redundant with the previous one; please let me know if there's some key info that was added by this para.